network-api/networkapi/nav/blocks.py

```diff
@@ -17,6 +18,8 @@ def open_in_new_window(self) -> bool:
 
 
 class NavLinkBlock(BaseLinkBlock):
+    description = blocks.CharBlock(required=False)
```

Comment:
Should we specify length limit to this field?

Reply:
Good catch! I've put this as 100 characters now to fit the largest description found on the mocks.